### DISCUSSION TITLE
Welcome to sierra-wireless-modems Discussions!

### DISCUSSION THREAD

**Discussion by danielewood** [danielewood/sierra-wireless-modems#56] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

**Comment by DaoudaK** [comment]
Hello, I have a Nokia 7705 SAR-Hm device including a Sierra Wireless modem (74xx generic firmware : SWI9X30C_02.33.03.00).
The device is  set to Band 38 but I'd rather like to set it  to Band 28.
I think this should be possible using AT commands.
Could somebody tell me where I can find the official set of  Sierra Wireless 74xx modem?
Thanks

BR

**Reply by mrizwan120** [reply]
how to change IMEI of my dell laptop WWAN of dw5811e snapdragon x7 lte pls can some body help me out

**Comment by mrizwan120** [comment]
how to change IMEI of my dell laptop WWAN of dw5811e snapdragon x7 lte pls can some body help me out

**Reply by adi8900** [reply]
why do you want to change imei lmao

**Comment by ankurjain23** [comment]
I am currently using Ubuntu 20.04 and I am facing an issue where my modems (USB modems MC7304 & MC7430)keep disconnecting and reconnecting frequently. I have installed MBPL drivers. Please help
